packages/wasm/src/services/blockchain/service.ts

```diff
@@ -0,0 +1,209 @@
+// @ts-nocheck
+
+import {DirectSecp256k1HdWallet} from '@cosmjs/proto-signing';
+import {CheqdAPI} from '@docknetwork/cheqd-blockchain-api';
+import {CheqdCoreModules} from '@docknetwork/cheqd-blockchain-modules';
+import {MultiApiCoreModules} from '@docknetwork/credential-sdk/modules';
+import {
+  CoreResolver,
+  DIDKeyResolver,
+  ResolverRouter,
+  UniversalResolver,
+  WILDCARD,
+} from '@docknetwork/credential-sdk/resolver';
+import {initializeWasm} from '@docknetwork/crypto-wasm-ts/lib/index';
+import {DockAPI} from '@docknetwork/dock-blockchain-api';
+import {
+  DockCoreModules,
+  DockDIDModule,
+} from '@docknetwork/dock-blockchain-modules';
+import {EventEmitter} from 'events';
+import {Logger} from '../../core/logger';
+import {once} from '../../modules/event-manager';
+import {utilCryptoService} from '../util-crypto';
+import {InitParams, validation} from './configs';
+
+let dockInstance;
+
+export function getDock() {
```

Comment:
no getCheqd?

Would a look up by name be more extensible in the future? e.g. getChainService(chainname)

Reply:
I should remove this function, we will not be using dock directly anymore